redaxo/src/addons/be_style/assets/css/minibar.css

```diff
@@ -0,0 +1,4 @@
+/*
+ *  Font Awesome 4.7.0 by @davegandy - http://fontawesome.io - @fontawesome
```

Comment:
sollten wir font-awesome dann jetzt aus  der styles.css rausnehmen, oder?

Reply:
SVGs wären sicherlich die bessere Alternative, aber dann müsste jeder der etwas für die Minibar liefert auch eines erst erstellen. So hätte man zumindest einen Pool von Icons. 

Ich lasse die FontAwesome Icons für die Minibar mit einem eigenen Prefix laden, so dass eine Kollision im Frontend vermieden wird.

siehe 5414e6f1ee62ce6450c69f13e3d84ce0866a1ae5

Wäre die Lösung in deinem Sinne ?

Reply:
Für den Anfang ist es „ok“ und soll kein show stopper sein.

Allgemein bin ich mittlerweile kein freund mehr von icon fonts, insofern man nur so wenige icons brauch wie wir aktuell. Der overhead ist schon beträchtlich.